File: python/7_kyu/indexed_capitalization.py
# Indexed capitalization
# https://www.codewars.com/kata/59cfc09a86a6fdf6df0000f1/train/python

def capitalize(s, ind):
    lis = list(s)
    results = []
    for i in range(0, len(lis)):
        if i in ind:
            results.append(lis[i].upper())
        else:
            results.append(lis[i])

    return ''.join(results)

File: javascript/7_kyu/indexed_capitalization.js
// Indexed capitalization
// https://www.codewars.com/kata/59cfc09a86a6fdf6df0000f1/train/javascript

function capitalize(s, arr) {
    var ans = [];
    for (var i = 0; i < s.length; i++) {
        if (arr.includes(i)) {
            ans.push(s[i].toUpperCase());
        } else {
            ans.push(s[i]);
        }
    }

    return ans.join('');

}

console.log(capitalize("abcdef",[1,2,5]));
console.log(capitalize("abcdef",[1,2,5,100]));